### PR TITLE
fix issue where events would come in as an array

### PR DESCRIPTION
**Pull request by gizm0duck**
running slackbutton_bot.js would crash on startup because events were coming in as an array already.

### CODE REVIEW

File: lib/CoreBot.js [hunk 1]
```diff
@@ -640,7 +640,11 @@ function Botkit(configuration) {
 
     botkit.on = function(event, cb) {
         botkit.debug('Setting up a handler for',event);
-        var events = event.split(/\,/g);
+        if (typeof(event) == 'string')
```
Comment (anonrig) [inline lib/CoreBot.js]: Can you put this line into a single if statement?

Comment (gizm0duck) [inline lib/CoreBot.js]: Just put onto a single line, let me know if that's acceptable.

**Comment by RafaelCosman** [comment]
Good catch, @gizm0duck! I think that I introduced this bug. It's good that `hears()` and `on()` will now both support both syntaxes.

Testing your PR now...

**Comment by RafaelCosman** [comment]
Just made this pass the linter: `5 passing (1s)`.

**Comment by RafaelCosman** [comment]
This works great. Thanks for fixing this, @gizm0duck. Merging now.

File: lib/CoreBot.js [hunk 2]
```diff
@@ -640,7 +640,8 @@ function Botkit(configuration) {
 
     botkit.on = function(event, cb) {
         botkit.debug('Setting up a handler for',event);
-        var events = event.split(/\,/g);
+          var events = (typeof(event) == 'string') ? event.split(/\,/g) : event;
```
Comment (anonrig) [inline lib/CoreBot.js]: Can you use `isArray` function? `Array.isArray(event)` is more readable.

**Pull request merged**